cmd/blox_new.go

```diff
@@ -23,13 +23,10 @@ var (
 // newCmd represents the new command
 var newCmd = &cobra.Command{
 	Use:   "new",
-	Short: "Create a new content file",
-	Long: `A longer description that spans multiple lines and likely contains examples
-and usage of using your command. For example:
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
+	Short: "Create a new content file for the target dataset",
+	Long: `This command will allow you to create new content based on the
+template attributes within the schemata. By providing a dataset name and ID
```

Comment:
maybe `dataset name and ID(slug)` to make the ID term more friendly to our usage.